README.md

```diff
@@ -6,8 +6,15 @@ The ESP32 detect the pulses outputing by a power meter and send it to an endpoin
 
 Currently the configurations are hardcoded. In the future you may change it to dynamic settings configuring over TCP or UART.
 
-For `WifiSsid`, `WifiPassword` and `Authkey`, you must create a file named `loginData.hpp`, which is ignored by git, and define those constants here.
-AuthKey can be  like `Bearer abc`.
+For `WifiSsid`, `WifiPassword` and `Authkey`, you must create a file named `loginData.hpp`, which is ignored by git:
```

Comment:
"For `WifiSsid`, `WifiPassword` and `Authkey` create a file ..."

Reply:
Der Pfad laut gitignore ist `main/loginData.hpp` - der sollte hier auch stehen.

Reply:
"create a file …, which is ignored by git"

Suggeriert, dass man das Ignorieren noch herstellen muss. Besser: "This path is in the gitignore."